static/app/views/replays/detail/focusArea.tsx

```diff
@@ -1,25 +1,39 @@
 import React, {useState} from 'react';
 
 import EventEntry from 'sentry/components/events/eventEntry';
-import {MemorySpanType} from 'sentry/components/events/interfaces/spans/types';
+import type {MemorySpanType} from 'sentry/components/events/interfaces/spans/types';
 import TagsTable from 'sentry/components/tagsTable';
-import {Event} from 'sentry/types/event';
+import type {BreadcrumbTypeConsole} from 'sentry/types/breadcrumbs';
+import type {Event} from 'sentry/types/event';
+import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import {useRouteContext} from 'sentry/utils/useRouteContext';
 
 import {TabBarId} from '../types';
 
+import Console from './console';
 import FocusTabs from './focusTabs';
 import MemoryChart from './memoryChart';
 
 type Props = {
+  consoleMessages: BreadcrumbTypeConsole[] | undefined;
   event: Event;
   eventWithSpans: Event | undefined;
   memorySpans: MemorySpanType[] | undefined;
 };
 
+const TABS = new Set(['console', 'performance', 'errors', 'tags', 'memory']);
+
+function isTabBarId(tab: string): tab is TabBarId {
+  return TABS.has(tab);
+}
+
 function FocusArea(props: Props) {
-  const [active, setActive] = useState<TabBarId>('performance');
+  const location = useLocation();
+  const hash = location.hash.replace(/^#/, '');
+  const tabFromHash = isTabBarId(hash) ? hash : 'performance';
+
+  const [active, setActive] = useState<TabBarId>(tabFromHash);
```

Comment:
This allows us to show the correct tab on page load.

Reply:
Oh, i didn't even deal with this case. :(

Let's put this in another PR. I regret splitting things up between `FocusArea` and `FocusTabs`, and now having both `location.hash` & `useState` probably isn't needed.